modular_skyrat/modules/verbs/code/subtle.dm

```diff
@@ -59,6 +59,10 @@
 
 	for(var/mob/receiver in viewers)
 		receiver.show_message(subtle_message, alt_msg = subtle_message)
+		// Optional sound notification
+		var/datum/preferences/prefs = receiver.client?.prefs
+		if(prefs && prefs.read_preference(/datum/preference/toggle/subtler_sound))
+			receiver.playsound_local(get_turf(receiver), 'sound/misc/bloop.ogg', 50)
```

Comment:
Would be nice to keep these consistent and play the `sound/effects/achievement/glockenspiel_ping.ogg`. Alternatively anything that isn't an admin announcement sound as that should be reserved due to its importance

Reply:
Yeh that's why I marked it DNTM. So nobody put it in before I changed the sound to match.

Originally the thought was so people cam tell between subtle and subtler, but I think I'll do that with a change in spans instead.